fastlane/Fastfile

```diff
@@ -428,4 +428,44 @@ platform :ios do
 
   end
 
+  desc "Generate release podspecs"
+  lane :generate_release_podspecs do |options|
+
+    xcframeworks_folder_path = options[:xcframeworks_folder_path]
+    podspec_name = options[:podspec_name] #e.g: GiniBankSDK or GiniBankSDKPinning
+    podspecs_repo_sdk_folder_path = options[:podspecs_repo_sdk_folder_path]
+
+    # Get the latest Git tag
+    latest_tag = sh("git describe --tags --abbrev=0").strip
+
+    # Extract the version from the tag (assuming the tag format is "GiniBankSDK;X.X.X")
+    version_folder = latest_tag.match(/;(\d+\.\d+\.\d+)/).captures.first
```

Comment:
[Request] Use `get_latest_version_from_release_tags(podspec_name, UI)` from `versions.rb`:
```suggestion
    # Extract the version from the release tag (assuming the tag format is "GiniBankSDK;X.X.X")
    version_folder = get_latest_version_from_release_tags(podspec_name, UI)
```

Reply:
@a-szotyori there is a reason why we are not adding a version tag also for everything with Pinning? like: `GiniBankSDKPinning,3.7.2`? As far as I can see we are adding just for the main SDKs.
If we want to use `get_latest_version_from_release_tags` instead of `latest_tag.match(/;(\d+\.\d+\.\d+)/).captures.first` we will need to start adding tags for every SDK including Pinning.

Reply:
We consider pinning be a part of the SDK release and always release both without and with pinning whenever we do an SDK release (you can see this in the `bank-sdk.release.yml` workflow).

You could remove the "Pinning" suffix from the `podspec_name` and pass that to the `get_latest_version_from_release_tags` function to get the version: 
```
# Remove "Pinning" suffix because we don't tag pinning releases separately (pinning version is released together with the non-pinning version)
project_id = podspec_name.sub('Pinning', '') 
get_latest_version_from_release_tags(project_id, UI)
```

Reply:
Done! Thanks!